pyproject.toml

```diff
@@ -2,14 +2,48 @@
 name = "baybe"
 description = "Bayesian Back End for Design of Experiments"
 authors = [
-    {name = "Martin Fitzner (Merck KGaA, Darmstadt, Germany)", email = "martin.fitzner@merckgroup.com"},
-    {name = "Adrian Šošić (Merck Life Science KGaA, Darmstadt, Germany)", email = "adrian.sosic@merckgroup.com"},
-    {name = "Alexander Hopp (Merck KGaA, Darmstadt, Germany)", email = "alexander.hopp@merckgroup.com"},
-    {name = "Alex Lee (EMD Electronics, Tempe, Arizona, USA)", email = "alex.lee@emdgroup.com"},
+    { name = "Martin Fitzner (Merck KGaA, Darmstadt, Germany)", email = "martin.fitzner@merckgroup.com" },
+    { name = "Adrian Šošić (Merck Life Science KGaA, Darmstadt, Germany)", email = "adrian.sosic@merckgroup.com" },
+    { name = "Alexander Hopp (Merck KGaA, Darmstadt, Germany)", email = "alexander.hopp@merckgroup.com" },
+    { name = "Alex Lee (EMD Electronics, Tempe, Arizona, USA)", email = "alex.lee@emdgroup.com" },
 ]
 readme = "README.md"
-license = {text = "Apache-2.0"}
-requires-python =">=3.8.6"
+license = { text = "Apache-2.0" }
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Operating System :: OS Independent",
+    "Typing :: Typed",
+]
+keywords = [
+    "Active Learning",
+    "Bayesian Optmization",
+    "Design of Experiments",
+    "DOE",
+]
+dynamic = ['version']
+
+[project.urls]
+"Documentation" = "https://emdgroup.github.io/BayBE/"
```

Comment:
no reason to wait, we just need to make sure we consistently se `BayBE` or `baybe` for the links

Reply:
Is this conversation the last thing we need to have before merging? Or are there any other open points?

Reply:
im not sure its so nice to have this github feature where it shows me the old outdated code. It seems this has been reoslved by choosing the case version. If we agree to the repo renaming to lowercase this is all good and consistent so this can be closed?